spyder/utils/syntaxhighlighters.py

```diff
@@ -1052,6 +1052,7 @@ class PygmentsSH(BaseSH):
     # Store the language name and a ref to the lexer
     _lang_name = None
     _lexer = None
+    _charlist = []

```

Comment:
This doesn't seem to be needed because `_charlist` is also initialized in `__init__` below.